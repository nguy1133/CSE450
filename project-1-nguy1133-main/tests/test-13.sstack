// If multiple items are on the stack, only the top one is printed each time.
PUSH "1st"
PUSH "2nd"
PUSH "3rd"
PRINT
PRINT
