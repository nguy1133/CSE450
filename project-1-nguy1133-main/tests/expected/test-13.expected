3rd
2nd
